src/prelude.fs

```diff
@@ -33,3 +33,10 @@ module internal Timer =
         t.Enabled <- true
         t.Start()
 #endif
+
+module AsyncHelpers =
```

Comment:
Any chance we can call it just 'Async`?

Reply:
Also `inline` is the only reason it should remain public, maybe we can remove that and make it `internal`?

Reply:
I originally didn't do it on purpose because I was afraid of shadowing the native API `Async.Start`

But it seems like the native API use `Start` and in our case we use `start` - capital `S` vs lower case `s`

However, I am still a little reluctant to do it, because it could be confusing the user, as if he `open Elmish` which is command then both version will be exposed and it is not obvious which one does what.

Reply:
If we mark it as `internal` then Elmish users will not be able to use it for creating their own `Cmd` helpers.

This is not too much of a concern for .NET, however for Fable it is problematic because they would need to recreate the helper which use `Timer` under the hood.